src/Manifolds.jl

```diff
@@ -45,6 +45,7 @@ import ManifoldsBase: base_manifold,
     inverse_retract!,
     isapprox,
     is_manifold_point,
+    injectivity_radius,
```

Comment:
Why is this in this place? There should be `injectivity_radius` a few lines above.

Reply:
Interesting, I must have missed that – we should maybe find a way to make this more structured (also the exports).

Reply:
It's mostly in alphabetical order but probably it should be in alphabetical order *exactly*.

Reply:
It should be on the main branch in the mean time, since I did that on another PR.